### PR TITLE
How it works default page

### LINKED ISSUE #3501: Add how it works page
Add the first page of how it works

### PR DESCRIPTION
**Pull request by master12**
Fixes #3501

[:sunglasses: PREVIEW](https://federalist-proxy.app.cloud.gov/preview/onrr/doi-extractives-data/how-it-works-default-page/)

Changes proposed in this pull request:

- Added main page for how it works
-
-

* Known issue for side nav

### CODE REVIEW

**Comment by master12** [comment]
The IE bug also exists in production.

**Comment by brentryanjohnson** [comment]
Links are 404s...are the preview env variables not resolving properly?

list of links about laws and regulations that display as broken

**Comment by brentryanjohnson** [comment]
The Native American section has Liquid data references that appear to have been commented out, leaving the section without the data-driven content.

Revenue - natural resources are increasingly a key source of income for many Native American tribes - then missing data for disbursements to tribes

```
<!--
{% assign disbursements = site.data.federal_disbursements.US["American Indian Tribes"].All %}
{% assign disbursement_year = site.data.years.disbursements | to_s | default: default_year %} -->
        <div>
          <h3 class="h3 landing-heading"><a href="/how-it-works/native-american-revenue/">Revenue</a></h3>
          <p>Natural resources are increasingly a key source of income for many Native American tribes. In <!--{{ disbursement_year }}-->,<glossary-term>ONRR</glossary-term> and <glossary-term>OST</glossary-term> disbursed <!-- {{ disbursements[disbursement_year] | intword | intcomma_dollar }}--> to tribes and allottees.</p>
```

File: gatsby-site/src/templates/how-it-works-default.js [hunk 1]
```diff
@@ -1,12 +1,41 @@
...
 		return (
 			<div>{hastReactRenderer(this.props.pathContext.markdown.htmlAst)}</div>
 		);
 	}
 }
-export default HowItWorksDefault;
\ No newline at end of file
+export default connect(
+  state => ({}),
+  dispatch => ({  hydateDataManager: (dataSets) => dispatch(hydateDataManagerAction(dataSets)),
```
Comment (brentryanjohnson) [inline gatsby-site/src/templates/how-it-works-default.js]: Is this typo for "hydrate" going to cause any issues, @master12?

**Comment by master12** [comment]
Closing this PR in favor of a new PR for this page and the native american ownership page

**Pull request closed without merging**